harness/determined/cli/master.py

```diff
@@ -11,64 +11,65 @@
 
 @authentication.required
 def config(args: Namespace) -> None:
-    if args.log:
-        if args.level and args.color:
-            if args.color.lower() == "on":
-                master_config = bindings.v1Config(
-                    log=bindings.v1LogConfig(level=args.level, color=True)
-                )
-            elif args.color.lower() == "off":
-                master_config = bindings.v1Config(
-                    log=bindings.v1LogConfig(level=args.level, color=False)
-                )
-            else:
-                raise CliError(
-                    "Error: Invalid value for --color provided: "
-                    + args.color
-                    + ". This optional parameter only accepts 'on' and 'off' values."
-                )
-            field_mask = ["log"]
-        elif args.level:
-            master_config = bindings.v1Config(log=bindings.v1LogConfig(level=args.level))
-            field_mask = ["log.level"]
-        elif args.color:
-            if args.color.lower() == "on":
-                master_config = bindings.v1Config(log=bindings.v1LogConfig(color=True))
-            elif args.color.lower() == "off":
-                master_config = bindings.v1Config(log=bindings.v1LogConfig(color=False))
-            else:
-                raise CliError(
-                    "Error: Invalid value for --color provided: "
-                    + args.color
-                    + ". This optional parameter only accepts 'on' and 'off' values."
-                )
-            field_mask = ["log.color"]
-        else:
-            raise CliError(
-                "Error: No log level or log color provided. Either log level or log"
-                + "  color is required to make changes to the master config log setting."
-            )
-        req = bindings.v1PatchMasterConfigRequest(
-            config=master_config, fieldMask=bindings.protobufFieldMask(paths=field_mask)
-        )
-        resp = bindings.patch_PatchMasterConfig(cli.setup_session(args), body=req).config
-    elif not args.log and (args.level or args.color):
+    resp = bindings.get_GetMasterConfig(cli.setup_session(args)).config
+    if args.json:
+        render.print_json(resp)
+    else:
+        print(util.yaml_safe_dump(resp, default_flow_style=False))
+
+
+def get_field_mask(args: Namespace) -> List[str]:
+    if args.log_color and args.log_level:
+        return ["log"]
```

Comment:
why do we need this for when they're both set? can't we just pass in `protobufFieldMask(paths=["log.color", "log.level"])` if both are set?